packages/apps/frontera/tests/pages/loginPage.ts

```diff
@@ -32,4 +32,8 @@ export class LoginPage {
     await this.page.click('#upgradeConsentCheckbox');
     await this.page.click('#idSIButton9');
   }
+
+  async login() {
+    await this.page.goto('http');
+  }
```

Comment:
is this goTo call ok?

Reply:
The test is in development and I provide the proper link when running from local. When time comes to have this test in the pipeline, it will grab the URL from secrets. The URL includes the token to bypass google\ms auth.

Reply:
so `http` is a token and not an actual url? Then it should be replaced with a more explicit name to prevent this kind of confusion.

Reply:
@acalinica , the text is updated now.